src/converters/fromZigbee.ts

```diff
@@ -5140,6 +5140,54 @@ const converters2 = {
             return payload;
         },
     } satisfies Fz.Converter,
+    honyer_metering: {
+        cluster: 'seMetering',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            if (meta.device.dateCode === '20170621') {
+                const result: KeyValueAny = {};
+                if (msg.data.hasOwnProperty('currentSummDelivered')) {
+                    const data = msg.data['currentSummDelivered'];
+                    const value = (parseInt(data[0]) << 32) + parseInt(data[1]);
+                    result.energy = value / 1000.0;
+                }
+                return result;
+            } else {
+                return converters1.metering.convert(model, msg, publish, options, meta);
+            }
+        },
+    } satisfies Fz.Converter,
+    honyer_electrical_measurement: {
+        cluster: 'haElectricalMeasurement',
+        type: ['attributeReport', 'readResponse'],
+        options: [
+            exposes.options.calibration('power', 'percentual'), exposes.options.precision('power'),
+            exposes.options.calibration('current', 'percentual'), exposes.options.precision('current'),
+            exposes.options.calibration('voltage', 'percentual'), exposes.options.precision('voltage'),
+        ],
+        convert: (model, msg, publish, options, meta) => {
+            if (meta.device.dateCode === '20170621') {
```

Comment:
If I understand correctly, there is a bug for this particular firmware which has the wrong multiplier/divisors?

Reply:
Yes, I tried the following method, but it didn't work.

electricityMeter({current: {divisor: 1000}, voltage: {divisor: 1}, power: {divisor: 1}}),

Reply:
Did you try to do a reconfigure afterwards? This should work (use the yellow refresh icon on the frontend to do so)

Reply:
Thanks for your guidance, the display is normal after using the yellow refresh button on the front.